package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@multiversx/sdk-nestjs-cli",
+  "version": "1.0.8",
```

Comment:
we should start from 1.0.0 when first publishing

Reply:
done